include/aikido/planner/TrajectoryPostProcessor.hpp

```diff
@@ -17,6 +17,13 @@ class TrajectoryPostProcessor
       const aikido::trajectory::InterpolatedPtr& _inputTraj,
       const aikido::common::RNG* _rng)
       = 0;
+
+  /// \param _inputTraj The untimed *spline* trajectory for the arm to process.
+  /// \param _rng Random number generator.
+  virtual std::unique_ptr<aikido::trajectory::Spline> postprocess(
+      const std::unique_ptr<trajectory::Spline>& _inputTraj,
```

Comment:
I don't believe `postprocess` wants to take the ownership of trajectories. Could we change this to just const reference type as well as the above `Interpolated` version? If this introduces too much work and out of this PR's scope, we could do this as a separate PR.

Reply:
Sure! These classes are pretty new, so changing the API shouldn’t break things too much.